crates/bevy_pbr/src/material.rs

```diff
@@ -316,7 +316,10 @@ where
                 .add_systems(
                     ExtractSchedule,
                     (
-                        extract_mesh_materials::<M>.before(ExtractMeshesSet),
+                        early_sweep_material_instances::<M>
+                            .before(late_sweep_material_instances)
+                            .before(ExtractMeshesSet),
+                        extract_mesh_materials::<M>.before(early_sweep_material_instances::<M>),
```

Comment:
Why not swap these two and chain their execution?

Reply:
Done.